README.md

```diff
@@ -61,4 +61,62 @@ client.on('messageCreate', message => {
         message.channel.createMessage(menu.build('Click this'));
     }
 });
-~~~
\ No newline at end of file
+~~~
+- Attachment - Local File
+~~~javascript
+const ErisAddons = require('eris-addons');
+const {Client} = require('eris');
+const client = new Client('bot token');
+
+client.on('messageCreate', message => {
+    if(message.content.toLowerCase() === '!attachment') {
+    const Attachment = new ErisAddons.Attachment()
+    Att.setFile(require.resolve('./example/file.png'))
+    Att.setName('file.png')
+
+    const embed = new ErisAddons.Embed();
+      embed.setAuthor(message.author.username, message.author.avatarURL);
+      embed.setTitle('Title of embed');
+      embed.setDescription('Description of embed');
+      embed.setColor('0x7289DA');
+      embed.setImage('attachment://file.png')
+      embed.setFooter(message.author.username, message.author.avatarURL);
+      embed.setTimestamp();
+    
+      message.channel.createMessage({embeds: [embed] }, Attachment);
+    }
+~~~
+- Attachment - Buffer 
+~~~javascript
+const ErisAddons = require('eris-addons');
+const {Client} = require('eris');
+const client = new Client('bot token');
+
+client.on('messageCreate', async message => {
+    if(message.content.toLowerCase() === '!attachment') {
+
+    const { createCanvas, loadImage } = require('canvas')
+    const canvas = createCanvas(720, 720)
+    const ctx = canvas.getContext('2d')
+    const img = await loadImage('Image URL');
+     
+    ctx.drawImage(img, 0, 0, canvas.width, canvas.height);
+    
+    //This is just an example, you can use any image editing package and any image buffer.
+
+    const Attachment = new ErisAddons.Attachment()
+    Att.setFile(canvas.toBuffer())
```

Comment:
edit this part, the variable name is wrong.
the same thing in the lines 74 and 75

Reply:
It works both ways, but I will edit it.